platform/linux-generic/include/odp_ipsec_internal.h

```diff
@@ -176,6 +178,10 @@ struct ipsec_sa_s {
 			unsigned	aes_ctr_iv : 1;
 			unsigned	udp_encap : 1;
 
+			/* Only for ESN */
```

Comment:
Please remove this comment and the preceding blank line. We do not have comments like "only for UDP encapsulation" for other flags either as such things are quite obvious. And the comment is not even accurate since the flags are used also without ESN (to see that ESN is off and to see that zero bytes of ESN gets inserted in the implicit trailer.